src/styles/_catalog.scss

```diff
@@ -1,10 +1,25 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
```

Comment:
that's what I see on 487px, if you start with a mobile-first markup, than here you should describe columns for mobile breakpoint, not for desktop one
<img width="567" alt="image" src="https://user-images.githubusercontent.com/37161315/157227961-12d56dca-cde6-46a3-9273-1dd52bfb6199.png">